sdk/node/lib/hash.js

```diff
@@ -0,0 +1,160 @@
+/**
```

Comment:
Please align the copyright with the template [here](https://github.com/hyperledger/fabric/blob/master/docs/dev-setup/headers.txt).